frontend/src/v5/ui/routes/viewer/theme.ts

```diff
@@ -314,6 +314,9 @@ export const theme = createTheme(
 								backgroundColor: COLOR.PRIMARY_MAIN,
 								color: COLOR.PRIMARY_MAIN_CONTRAST,
 								textDecoration: 'none',
+								'&.Mui-disabled': {
```

Comment:
When you a TabContainer and any of those tabs is disabled

Reply:
![image](https://user-images.githubusercontent.com/43547703/174338419-ca9df63c-62a4-4be8-8c22-3b0ecd917e42.png)

This is an example of the modal in the process of submitting a request. As you can see, the tabs are disabled to prevent a tab change until the submission request isn't complete

Reply:
I see. Does this need to go in the main theme file not the viewer theme?

Reply:
Sorry, no, that was for the submit button in the viewer which, when disabled, still looked active. This code in the viewer must be a leftover, I'll remove it.